sfeir-open-source/talk-control
# Add slave implementation

### Expected behavior
You should be able to control the demo slideshow from the master.
### How to reproduce
- Start the app: `npm start`
- Open your browser on http://localhost:1234
- Validate the input with the default value

You should now be able to interact with arrows and spacebar.
___
Closes #26 , #16
